androguard/core/bytecodes/axml/__init__.py

```diff
@@ -1084,12 +1084,12 @@ def _fix_value(self, value):
         if not self.__charrange or not self.__replacement:
             if sys.maxunicode == 0xFFFF:
```

Comment:
Do you know if there are narrow 3.x builds? otherwise we can remove this too...

Reply:
I don't know.